vmhost/vmhooks/baseOps.go

```diff
@@ -2076,11 +2089,16 @@ func StorageLoadWithHost(host vmhost.VMHost, keyOffset int32, keyLength int32, d
 func StorageLoadWithWithTypedArgs(host vmhost.VMHost, key []byte) ([]byte, error) {
 	storage := host.Storage()
 	metering := host.Metering()
-	data, usedCache, err := storage.GetStorage(key)
+	data, trieDepth, usedCache, err := storage.GetStorage(key)
 	if err != nil {
 		return nil, err
 	}
-	storage.UseGasForStorageLoad(storageLoadName, metering.GasSchedule().BaseOpsAPICost.StorageLoad, usedCache)
+
+	err = storage.UseGasForStorageLoad(storageLoadName, int64(trieDepth), metering.GasSchedule().BaseOpsAPICost.StorageLoad, usedCache)
+	if err != nil {
+		return []byte{}, err
```

Comment:
is it ok to return here empty byte slice? On v1.5 it returned nil.

Reply:
Changed to nil.